docs/faq/code-analysis/does-codacy-place-limits-on-the-code-analysis.md

```diff
@@ -43,20 +45,30 @@ Some tools fail to calculate duplication or time out when analyzing a number of
 See <a href="../../troubleshooting/why-arent-duplication-metrics-being-calculated/">Why aren't duplication metrics being calculated?</a>
 </td>
 </tr>
+
 <tr>
 <td>Number of issues per file and per tool</td>
 <td>50</td>
 <td>
 Codacy limits the number of issues returned on each file by individual tools as a safeguard against degradation of performance on large or unexpected use cases. This means that in some situations Codacy could report more issues after a push that includes fixes for the currently reported issues.
 </td>
 </tr>
+
```

Comment:
The fact that you added this blank line on your pull request gave me the idea of [separating each table row](https://github.com/codacy/docs/pull/1238/commits/3c77b7e49dd3047f998ff9b0ee6f108a8bea5555) to aid the readability of the HTML in the source file. :+1: